compass/ocean/tests/internal_wave/rpe_test/__init__.py

```diff
@@ -11,7 +11,7 @@ class RpeTest(TestCase):
     5 different values of the viscosity at the given resolution.
     """
 
-    def __init__(self, test_group):
+    def __init__(self, test_group, vlr=False):
```

Comment:
Again, needs to be added to the docstring.

Reply:
Addressed by https://github.com/MPAS-Dev/compass/pull/564/commits/74755efd7497e3937d080aeea63ce0400564cf4d